services/monitoring/prometheus/prometheus-aws.yml

```diff
@@ -0,0 +1,38 @@
+scrape_configs:
+  - job_name: 'federate-computational-clusters-ec2'
+    honor_labels: true
+    metrics_path: '/federate'
```

Comment:
very nice, bu we should have basic auth nevertheless there as well

Reply:
yes, do you have experiment on how to add the basic auth stuff?

Reply:
and the auth I guess will need to be set into osparc-simcore as well

Reply:
as discussed it needs 2 PRs:
will do one now in osparc-simcore to setup a password